src/main/java/org/cactoos/func/CallableOf.java

```diff
@@ -66,39 +58,42 @@
      * @since 0.32
      */
     public CallableOf(final Runnable runnable, final T result) {
-        this(new FuncOf<>(runnable, result));
+        this(() -> {
+            runnable.run();
+            return result;
+        });
     }
 
     /**
      * Ctor.
      * @param proc Encapsulated proc
+     * @param ipt Input
      * @param result Result to return
      * @since 0.32
      */
-    public CallableOf(final Proc<X> proc, final T result) {
-        this(new FuncOf<>(proc, result));
+    public CallableOf(final Proc<X> proc, final X ipt, final T result) {
+        this(new FuncOf<>(proc, result), ipt);
     }
 
     /**
      * Ctor.
      * @param fnc Encapsulated func
+     * @param ipt Input
      */
-    public CallableOf(final Func<X, T> fnc) {
-        this(fnc, null);
+    public CallableOf(final Func<X, T> fnc, final X ipt) {
+        this(() -> fnc.apply(ipt));
     }
 
     /**
      * Ctor.
-     * @param fnc Encapsulated func
-     * @param ipt Input
+     * @param cal Encapsulated callable
      */
```

Comment:
Consider adding a `@since` tag to be consistent with other constructors.

Reply:
@victornoel This constructor is still missing the `@since` tag.

Reply:
@vzurauskas yep, sorry, now it's ok :)